apps/server/src/modules/learnroom/service/dashboard.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain/types';
+import { IDashboardRepo, DashboardElementRepo } from '@shared/repo';
+
+@Injectable()
+export class DashboardService {
+	constructor(
+		@Inject('DASHBOARD_REPO') private readonly dashboardRepo: IDashboardRepo,
+		private readonly dashboardElementRepo: DashboardElementRepo
+	) {}
+
+	async deleteDashboardByUserId(userId: EntityId): Promise<number> {
+		const usersDashboard = await this.dashboardRepo.getUsersDashboard(userId);
+		await this.dashboardElementRepo.deleteByDashboardId(usersDashboard.id);
+		const deletedUsersDashboard = await this.dashboardRepo.deleteDashboardByUserId(userId);
+
+		return deletedUsersDashboard;
+	}
```

Comment:
Look here, the return in line 12 is number but name of return variable is "deletedUsersDashboard" why? I think name like "result" is better.

Reply:
ok